docs/docs/cmd/tenant/people/people-profilecardproperty-list.mdx

```diff
@@ -0,0 +1,108 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# tenant people profilecardproperty list
+
+Lists all profile card properties
+
+## Usage
+
+```sh
+m365 tenant people profilecardproperty list [options]
+```
+
+## Options
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be either **Tenant Administrator** or **Global Administrator**.
+
+:::
+
+## Examples
+
+Lists all profile card properties
+
+```sh
+
+```sh
+m365 tenant people profilecardproperty list
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+  "value": [
+    {
+      "directoryPropertyName": "customAttribute1",
+      "annotations": [
+        {
+          "displayName": "Cost center",
+          "localizations": [
+            {
+              "languageTag": "de",
+              "displayName": "Kostenstelle"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "directoryPropertyName": "Alias",
+      "annotations": []
+    }
+  ]
+}
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  directoryPropertyName  displayName     displayName de
+  ---------------------  --------------  --------------
+  customAttribute1       Cost center     Kostenstelle
+  Alias
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  directoryPropertyName,displayName,displayName de
+  customAttribute1,Cost center,Kostenstelle
+  Alias,,
+  ```
```

Comment:
If I'm not mistaken, for `CSV` output, it doesn't return `displayName` anymore so we can remove it here.

```suggestion
  ```csv
  directoryPropertyName
  customAttribute1
  Alias
  ```
```

Reply:
nope that's not true. The version you show is only present when we display output that does not have any customattributes. BUT when we do have some custom properties defined then those have display name and could have display name for localization as well

![image](https://github.com/pnp/cli-microsoft365/assets/58668583/0e7443a3-9381-4246-b414-3ae86a9e0f36)

Reply:
I'm getting this

![image](https://github.com/pnp/cli-microsoft365/assets/38426621/8e863d4e-c3d4-47ed-861c-3e5c176516fc)

Reply:
And as you can see, I've quite a few customAttributes from testing 😅